index.d.ts

```diff
@@ -0,0 +1,24 @@
+import {type Buffer} from 'node:buffer';
```

Comment:
One note with this import: this means `strip-final-newline` will not be able to be imported from a browser context anymore, when using TypeScript, I believe.

Fundamentally, this is because this module supports `Buffer`, which is Node.js-specific. It turns out the current code is not referencing `Buffer` directly though, but the types might need to.

Please let me know your thoughts on this.

Reply:
You can drop `Buffer` here. It wil still be supported because of `Uint8Array`.

Reply:
Ok, fixed :+1: 

I have kept `Buffer` in the document and the type tests, but the tests themselves now do not use it, which means it should work in browsers.